package.json

```diff
@@ -31,7 +31,8 @@
     "typecheck": "pnpm --recursive typecheck"
   },
   "engines": {
-    "node": "^18.0.0 || >=20.0.0"
+    "node": "^18.0.0 || >=20.0.0",
+    "pnpm": "8.15.7"
```

Comment:
Do you think this should also be a range?

Reply:
I changed it to `>=8 <9` which should allow for some leeway.